### ISSUE TITLE
[4.0] Wrong aria-hidden value in menu with sub-items

### ISSUE THREAD

**Issue by drmenzelit** [joomla/joomla-cms#32519]
### Steps to reproduce the issue
Create a menu with sub-items.
Create a menu module on the sidebar (standard layout). Set the parameter "Sub-menu Items" to "Show".
Inspect the code of the menu.

### Expected result
The attribute "aria-hidden" should have the value "false"

### Actual result
The attribute "aria-hidden" has the value "true" although the sub-menu is open and visible.

### Additional comments
The Javascript controlling the menu should be checked.

**Comment by brianteeman** [comment]
#32068

**Comment by drmenzelit** [comment]
I knew we had it already :-)

Closing this because duplicate.
